### PR TITLE
provider/kubernetes: Make s3 persistence easier to set up

### PR DESCRIPTION
**Pull request by lwander**
@Cocotton FYI

@duftler PTAL

### CODE REVIEW

**Comment by Cocotton** [comment]
Thanks for the PR

File: experimental/kubernetes/config/front50-local.yml [hunk 1]
```diff
@@ -15,9 +15,12 @@ spinnaker:
...
     jsonPath: /root/.kube/account.json # Don't change this
-    rootFolder: # Folder to store artifacts in (will be created if necessary)
-    bucket: # Bucket to store artifacts in (will be created if necessary)
-
+    rootFolder: ${services.front50.bucket_root}
+    bucket: ${services.front50.bucket}
+  s3:
+    enabled: ${services.front50.s3.enabled}
+    rootFolder: ${services.front50.bucket_root}
+    bucket: ${services.front50.bucket}
```
Comment (Cocotton) [inline experimental/kubernetes/config/front50-local.yml]: Replace with: `${services.front50.storage_bucket}`

File: experimental/kubernetes/config/front50-local.yml [hunk 2]
```diff
@@ -15,9 +15,12 @@ spinnaker:
...
-    enabled: true
-    project: # Name of your GCP project
+    enabled: ${services.front50.gcs.enabled}
+    project: ${services.front50.gcs.project}
     jsonPath: /root/.kube/account.json # Don't change this
-    rootFolder: # Folder to store artifacts in (will be created if necessary)
-    bucket: # Bucket to store artifacts in (will be created if necessary)
-
+    rootFolder: ${services.front50.bucket_root}
+    bucket: ${services.front50.bucket}
```
Comment (Cocotton) [inline experimental/kubernetes/config/front50-local.yml]: Replace with: `${services.front50.storage_bucket}`

**Comment by Cocotton** [comment]
After replacing `${services.front50.bucket}` with `${services.front50.storage_bucket}`, it works like a charm!

File: experimental/kubernetes/README.md [hunk 3]
```diff
@@ -116,29 +116,38 @@ mentioned above.
...
-If you'd rather use S3, follow the guidelines
-[here](http://www.spinnaker.io/docs/front50-cassandra-to-s3).
+  front50:
+    host: spin-front50.spinnaker.svc.cluster.local
+    port: 8080
+    baseUrl: ${services.default.protocol}://${services.front50.host}:${services.front50.port}
+
+    # If using storage bucket persistence (gcs or s3), specify the bucket here
+    # disable cassandra and enable the storage service below.
+    storage_bucket: # Needs to be a unique name. Pick something clever.
```
Comment (duftler) [inline experimental/kubernetes/README.md]: Unique to the user's project? Globally unique?

**Comment by duftler** [comment]
LGTM once addressing comments.

**Pull request merged**